src/Apollo_Client/react/hooks/Apollo_Client__React_UseQuery.re

```diff
@@ -0,0 +1,99 @@
+module Core = Apollo_Client__Core;
+module Errors = Apollo_Client__Errors;
+module Graphql = Apollo_Client__Graphql;
+module GraphqlTag = Apollo_Client__GraphqlTag;
+module React_Types = Apollo_Client__React_Types;
+
+type simpleQueryResult('a) =
+  | Data('a)
+  | Error(Errors.ApolloError.t)
+  | Loading
+  | NoData;
+
+module Raw = {
+  [@bs.module "@apollo/client"]
+  external useQuery:
+    (
+      ~query: Graphql.Language.documentNode,
+      ~options: React_Types.QueryHookOptions.t('raw_tData, 'raw_tVariables)=?
```

Comment:
I think `jsData` could be easier to read and grasp for future contributors reading this

Reply:
Agreed.